selfdrive/car/ford/fordcan.py

```diff
@@ -133,7 +133,9 @@ def create_acc_msg(packer, CAN: CanBus, long_active: bool, gas: float, accel: fl
     "AccBrkTot_A_Rq": accel,                          # Brake total accel request: [-20|11.9449] m/s^2
     "Cmbb_B_Enbl": 1 if long_active else 0,           # Enabled: 0=No, 1=Yes
     "AccPrpl_A_Rq": gas,                              # Acceleration request: [-5|5.23] m/s^2
+    "AccPrpl_A_Pred": gas,                            # Acceleration request: [-5|5.23] m/s^2
```

Comment:
If it's like Hyundai's raw and filtered accel, we should probably check this in the safety, do you know what it does @haraschax? `AccPrpl_A_Rq` should be gas, `AccBrkTot_A_Rq` should be brake.

Reply:
https://github.com/commaai/panda/pull/1776